### DISCUSSION TITLE
Another Vercel account is using this domain.

### DISCUSSION THREAD

**Discussion by donah123** [vercel/community#6054] [Help] [answered]
### Page to Investigate

domain page

### Steps to Reproduce

I'm getting this error, and currently there are no domain added on the domain page. This was working before, we alter the domain and to put the domain back is now an issue. Can you help with this?.
I have already added the TXT record on my domain registrar, but still nothing has happened.

**Comment by amyegan** [comment, marked as answer]
Hey @donah123. For [domain access verification](https://vercel.com/docs/projects/domains/add-a-domain#verify-domain-access) via TXT to work, make sure you're using your registrar's nameservers and connect the domain to your Project with an A record. It can take up to 48 hours DNS changes to propagate globally.
